app/controllers/internal/tags_controller.rb

```diff
@@ -23,6 +24,8 @@ def update
     add_moderator if @add_user_id
     remove_moderator if @remove_user_id
     @tag.update!(tag_params)
+
+    notify(:internal, current_user, __method__) { tag_params.dup }
```

Comment:
I noticed that currently when a moderator is added or removed, 2 log records will be created. Is that intentional?
From the technical perspective it's valid, but may be confusing for people :thinking: 
Ideally, I would separate `add_moderator` and `remove_moderator` to separate actions (not as a part of this pr).
Maybe it's worth to create an `update` log only when there are no `@add_user_id` or `@remove_user_id`, what do you think?